scripts/main.js

```diff
@@ -35,4 +40,85 @@ document.addEventListener('DOMContentLoaded', () => {
     });
   });
   returnHome.addEventListener('click', disposeMobileMenu);
-});
\ No newline at end of file
+
+  /* Portfolio: details popup window */
+
+  const hideModal = () => {
+    recentWorkModal.classList.add('display-none');
+    recentWorkModal.innerHTML = `<div class="container"> 
+    <span id="close-modal-desktop"></span> 
+    <div id="feature"> <span id="close-modal">
+    </span> </div> </div>`;
+    fullPage.setAttribute('style', 'height: auto');
+  };
+
+  // this method will create and display modal from previously defined objects(project)
+  const showModal = (project) => {
+    const featureImage = document.createElement('img');
+    const title = document.createElement('h2');
+    const technology = document.createElement('div');
+    const technoList = document.createElement('ul');
+    const description = document.createElement('p');
+    const links = document.createElement('div');
+    const dispose = recentWorkModal.querySelector('#close-modal');
+    const disposeDesktop = recentWorkModal.querySelector('#close-modal-desktop');
+    links.innerHTML = `
+<a href="${project.liveVersion || '#'}" target="blank">
+  <span>See Live </span
+  ><img src="./images/icons/go-live.svg" alt="see live" />
+</a>
+<a href="${project.source || '#'}" target="blank">
+  <span>See Source </span>
+  <img src="./images/icons/GitHub-white.svg" alt="see source" />
+</a>`;
```

Comment:
- [ ] These popup modal buttons should be well aligned to the right and have some spacing around. In your case, the buttons aren't well aligned on the desktop version.

Your Design|
:----------------------:|
![a1](https://user-images.githubusercontent.com/39523312/156273266-c4634ba1-ea75-46e1-81f8-ed026e10c15e.png)

Template Design |
:------------------------:|
![a3](https://user-images.githubusercontent.com/39523312/156273314-2d2d9de3-9e3b-49df-8906-81a2f581fd84.png)

Reply:
@mmsesay great catch. I will defiantly address it. Thank you